runtime-modules/content/src/lib.rs

```diff
@@ -1666,7 +1666,17 @@ decl_module! {
         }
 
         /// Issue NFT
-        #[weight = 10_000_000] // TODO: adjust weight
+        ///
+        /// <weight>
+        ///
+        /// ## Weight
+        /// `O (1)`
```

Comment:
Should be `O(W + B)`

Reply:
The weight should be `O(W + B)` and DB should be `O(W)` (currently it's the other way around)